src/renderer/dx/DxRenderer.cpp

```diff
@@ -1322,8 +1318,8 @@ try
             // And persist the new size.
             _displaySizePixels = clientSize;
 
-            // Mark this as the first frame on the new target. We can't use incremental drawing on the first frame.
-            _firstFrame = true;
+            _invalidMap.resize(clientSize / _fontRenderData->GlyphCell());

```

Comment:
I'm confused -- the body of the PR says "responsibility for resizing `_invalidMap` has been moved entirely into `InvalidateAll`", but both places we have InvalidateAll calls we resize the map _before_ calling it?

Reply:
Woops I mixed things up. I'm fixing my PR body to say that responsibility for full invalidation was moved there.